admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/data/mapper/dateMapper.kt

```diff
@@ -1,8 +1,17 @@
 package net.thechance.mena.admin_panel.data.mapper
 
 import kotlinx.datetime.LocalDate
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.LocalTime
 
 fun parseLocalDateOrDefault(value: String?, defaultDate: LocalDate = DEFAULT_DATE) = value?.let {
     runCatching { LocalDate.parse(it) }.getOrDefault(defaultDate)
 } ?: defaultDate
-private val DEFAULT_DATE = LocalDate(9999, 12, 31)
\ No newline at end of file
+
+fun parseLocalDateTimeOrDefault(value: String?, defaultDateTime: LocalDateTime = DEFAULT_DATE_TIME) = value?.let {
+    runCatching { LocalDateTime.parse(it) }
+        .getOrDefault(DEFAULT_DATE_TIME)
+}
```

Comment:
this function should return default if value is null
```suggestion
fun parseLocalDateTimeOrDefault(value: String?, defaultDateTime: LocalDateTime = DEFAULT_DATE_TIME) = value?.let {
    runCatching { LocalDateTime.parse(it) }
        .getOrDefault(DEFAULT_DATE_TIME)
} ?: defaultDateTime
```